lib/taskana-core-test/src/test/java/acceptance/classification/query/QueryClassificationAccTest.java

```diff
@@ -0,0 +1,984 @@
+    defaultTestClassification()
+        .customAttribute(CUSTOM_1, testIdentifier)
+        .name("Classification Name")
+        .buildAndStoreAsSummary(classificationService);
+    defaultTestClassification()
+        .customAttribute(CUSTOM_1, testIdentifier)
+        .type("DOCUMENT")
+        .buildAndStoreAsSummary(classificationService);
+    defaultTestClassification()
+        .customAttribute(CUSTOM_1, testIdentifier)
+        .type("TASK")
+        .isValidInDomain(false)
+        .buildAndStoreAsSummary(classificationService);
+
+    List<String> columnNameList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .listValues(NAME, null);
+    List<String> columnTypeList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .listValues(TYPE, null);
+    List<String> columnTypeMasterList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .domainIn("")
+            .listValues(TYPE, null);
+    List<String> columnCreatedList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .domainIn("")
+            .listValues(CREATED, null);
+    DateTimeFormatterBuilder dateTimeFormatterBuilder =
+        new DateTimeFormatterBuilder()
+            .appendOptional(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS"))
+            .appendOptional(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SS"))
+            .appendOptional(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.S"));
+    List<String> columnNotValidInDomainMasterList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .domainIn("")
+            .validInDomainEquals(false)
+            .listValues(VALID_IN_DOMAIN, null);
+
+    assertThat(columnNameList).containsExactlyInAnyOrder("Classification Name", null);
+    assertThat(columnTypeList).containsExactlyInAnyOrder("TASK", "DOCUMENT", null);
+    assertThat(columnTypeMasterList).containsExactlyInAnyOrder("TASK", "DOCUMENT", null);
+    assertThat(columnCreatedList)
+        .isNotEmpty()
+        .allSatisfy(
+            c ->
+                assertThat(now)
+                    .isBefore(
+                        LocalDateTime.parse(c, dateTimeFormatterBuilder.toFormatter())
+                            .toInstant(ZoneOffset.UTC)));
+    // Expecting "0" for H2 (==> false in Oracle DBs) and "f" for POSTGRES (==> abbreviation of
+    // false)
+    assertThat(columnNotValidInDomainMasterList).containsAnyOf("f", "0");
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassifications_When_QueryingCategoryAndDomain() throws Exception {
```

Comment:
What do you think about should_FindClassifications_When_QueryingCategoryInAndDomainIn() ?

Reply:
I changed it to should_FindClassifications_When_QueryingForCategoryInAndDomainIn() according to other test names. What do you think about this?